### DISCUSSION TITLE
Unable to change "Recent" header on homepage with showRecent enabled

### DISCUSSION THREAD

**Discussion by nanowatts** [jpanther/congo#815] [Q&A] [answered]
My initial Hugo implementation was ~2y ago so I've been trying to get current recently.

The last minor issue that I'm encountering is driving me nuts. I use homepage.showRecent in params.toml to showcase posts on my homepage. Before upgrading to latest Hugo and Congo, I had an edit to the partial to change the style and text of this.

```
{{ if .Site.Params.homepage.showRecent | default false }}
  <h2 class="mt-8 text-2xl font-semibold"> Latest Posts </h2>
  {{ range first 5 (.Paginate (where .Site.RegularPages "Type" "in" .Site.Params.mainSections) 5).Pages }}
    {{ partial "article-link.html" . }}
  {{ end }}
{{ end }}
```

However, I can't seem to override "Recent" in extrabold being the style and text that is displayed on my homepage. I've poured over every reference for this and am coming up empty. Any ideas?

**Comment by wolfspyre** [comment]
is your repo and site available someplace we can take a peek at it? 
no promises, but I might be able to offer a bad idea or two. ;)

**Reply by nanowatts** [reply]
Yeah! Repo is [here](https://github.com/nanowatts/nanowatts.net) and live site is [here](https://nanowatts.net).

**Reply by wolfspyre** [reply, marked as answer]
looks like this is the file yer wanting to override:
https://github.com/nanowatts/nanowatts.net/blob/5fff4be5d8d93711ebdb83deb88f2bf116b2b04b/themes/congo/layouts/partials/recent-articles.html#L2

so... create a 'layouts' folder in the root of your project....

follow the directory structure of congo's `layouts`. 

put  the file you're trying to modify in your layouts folder.

That will tell hugo to use your version of the layout/partial instead of congo's version[^1].

that bit in and of itself may be sufficient to resolve the problem you're having...

if I grok correctly...  something like this might help?

```
mkdir layouts/partials/
cp themes/congo/layouts/partials/recent-articles.html layouts/partials
sed -ie 's/font-semibold/font-bold/' layouts/partials/recent-articles.html
echo "I will probably remember to examine commands being entered from the internet before executing them" >> layouts/partials/recent-articles.html
```
^^----  you may want to eyeball this.


[^1]: (side benefit: upstream synchronization becomes a buttload easier)

**Reply by nanowatts** [reply]
That did it! Much appreciated.
